data.sql

```diff
@@ -1,5 +1,4 @@
-/* Populate database with sample data. */
-
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals VALUES  (1, 'Agumon', 'Feb 3, 2020', 10.23, 'true', 0 ),
```

Comment:
- [ ] @Salwa99 great work on your queries, however, you forgot to indicate the columns for the values, this leads to some queries giving false outputs eg `SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;`, this is because the database is confusing the weight column with the escape attempts as shown in the screenshot below, kindly correct the error and also update the screenshot.


![image](https://user-images.githubusercontent.com/61408860/222076007-b36619aa-171a-4398-bf10-44bd82b7cb6c.png)

Reply:
I'm gonna fix it, thanks!